src/components/views/rooms/EditMessageComposer.js

```diff
@@ -176,16 +180,16 @@ export default class EditMessageComposer extends React.Component {
         const editedEvent = this.props.editState.getEvent();
         const editContent = createEditContent(this.model, editedEvent);
         const newContent = editContent["m.new_content"];
-        if (!this._isModifiedOrSameAsOld(newContent)) {
-            return;
+
+        if (this._isModifiedOrSameAsOld(newContent)) {
+            const roomId = editedEvent.getRoomId();
+            this._cancelPreviousPendingEdit();
+            this.context.matrixClient.sendMessage(roomId, editContent);
```

Comment:
wouldn't we not want to send the message at all?

Reply:
Yup, it only sends if `isModified`
don't know why the function has `orSameAsOld` in the name, it makes no sense to me, previously it said `if !isModified bail`

Reply:
well, the function does check to see if the old content is the same when it sees that there's been a modification, so the function name does make some amount of sense. Would be good to add a comment, give it a better name, or add docs overall to clarify what the function/branch is doing here.

Reply:
Should have probably been `_isModifiedOrDifferentFromOld`